documentation/modules/upgrading/con-upgrade-paths.adoc

```diff
@@ -0,0 +1,50 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='con-upgrade-paths-{context}']
+= Strimzi upgrade paths
+
+[role="_abstract"]
+Two Strimzi upgrade paths are possible:
+
+Incremental::
+Upgrading Strimzi from the previous minor version to version {ProductVersion}.
+
+Multi-version::
+Upgrading Strimzi from an old version to version {ProductVersion} within a single upgrade (skipping one or more intermediate versions).
++
+For example, upgrading from Strimzi 0.24.0 directly to Strimzi {ProductVersion}.
+
+[id='con-upgrade-paths-kafka-versions-{context}']
+== Supported Kafka versions
```

Comment:
Just a suggestion ... I think this should be a separate reference module ( instead of being part of a concept module)

Reply:
Quite possibly. We're trying not to split the content too much here. I'm going to leave it for now